gammapy/image/utils.py

```diff
@@ -260,11 +282,13 @@ def exclusion_distance(exclusion):
 
     Parameters
     ----------
-    TODO
+    exclusion : array_like
```

Comment:
By the way: what does the "~" in "~numpy.ndarray" mean? Approx. a numpy ndarray?